Add documentation, updated copyright (2018), and canonicalization

matthewzring opened the pull request.

Commit:
Update copyright (2018)

Commit:
Settings and welcome

Commit:
Canonicalization

Commit:
Documentation

TheMonitorLizard commented:
Looks good, will merge soon, just gotta work on being easily prepared for it.

Shengaero commented on README.md:
````diff
@@ -51,7 +51,7 @@ If you think you want to contribute to the bot, make sure to visit the
 ## License
 ```
-   Copyright 2017 Kaidan Gustave
+   Copyright 2017-2018 Kaidan Gustave
````
I would like you to revert this copyright change.
This is an action reserved only for the licenseholder, which you are not.

Commit:
:/

My apologies for the miscommunication. I will try to get in touch with you before working on something.
